Extending/Packages/package-types.md

```diff
@@ -0,0 +1,65 @@
+---
+versionFrom: 7.0.0
+---
+
+# Package Types
+There are two main ways to install packages in Umbraco, 
+package zip files and NuGet packages, each has its own advantages. 
+
+When developing a package you should attempt to provide both a zip file package and a NuGet package.
+
+## Package zip
+
+A package zip file can be installed through the Umbraco backoffice 
+
+![Zip packages can be installed via the Umbraco backoffice package section](images/backoffice-package-section.png)
+
+Packages zip files can contain: 
+
+* Solutions files (dlls, App_plugins files, etc)
+* Document Types
+* Templates
+* Stylesheets
+* Macros
+* Languages
+* Dictionary Items
+* Data Types
+
+:::note
+Currently you cannot include media items in a Package Zip file. You will need a custom [package action](package-actions.md) to install media as part of a package.
```

Comment:
When Jespers PR #2064 gets merged, this link needs to be updated, right? 
 
```
[package action](Package-Actions)
```

Reply:
And the same goes for the link in line 32 - I can see that's the same at least!

Reply:
yep I've updated them now, hopefully that's the right page.